apps/server/src/modules/tool/uc/dto/context-external-tool.types.ts

```diff
@@ -7,4 +7,5 @@ export type ContextExternalToolQuery = {
 	id?: string;
 	schoolToolRef?: Partial<SchoolExternalToolRefDO>;
 	contextId?: ContextExternalTool['contextId'];
```

Comment:
Would it make sense to combine `contextId` and `contextType` to a (optional) `context` property? That way we could treat them both as a unit. Would maybe make sense when one cannot exist without the other.